raiden-dapp/tests/unit/components/transfer/transfer-menus.spec.ts

```diff
@@ -62,5 +72,49 @@ describe('TransferMenus.vue', () => {
     );
   });
 
-  test('deposits successfully', async () => {});
+  test('deposit button opens deposit dialog', async () => {
+    const wrapper = createWrapper(One);
+    const depositButton = wrapper.find(
+      '.transfer-menus__deposit-channels__menu--deposit'
+    );
+
+    depositButton.trigger('click');
+    await wrapper.vm.$nextTick();
+
+    const channelDepositDialog = wrapper.find('.channel-deposit');
+
+    expect(channelDepositDialog.exists()).toBe(true);
+  });
+
+  test('clicking change token button displays token overlay', async () => {
+    const wrapper = createWrapper(One);
+    const tokenSelectButton = wrapper.findAll('span').at(0);
+
+    tokenSelectButton.trigger('click');
+    await wrapper.vm.$nextTick();
+
+    const tokenOverlay = wrapper.find('.v-overlay--active');
```

Comment:
Looks pretty generic. If we end up with multiple overlays that can be triggered in this menu, it won't work anymore. Why not using `wrapper.findComponent()`? Same for the dialog above.